Core/clim-basic/extended-streams/stream-input.lisp

```diff
@@ -393,6 +393,10 @@ keys read."))
 (defmethod stream-unread-char ((stream standard-extended-input-stream) char)
   (stream-unread-gesture stream char))
 
+(defmethod stream-peek-char ((stream standard-extended-input-stream))
+  (let ((ch (stream-read-gesture stream :peek-p t)))
+    (and (characterp ch) ch)))
+
 
 ;;; stream-read-gesture on string strings. Needed for accept-from-string.
```

Comment:
Looks like two empty lines.

The following comment (from a different commit) looks wrong. Should that have been "string streams"?

Reply:
comment fixed. re empty lines, it is page break (in emacs it shows as ^L)